add: skydiver/preset-nextjs (#2)
use-preset/awesome-presets: README.md

- [Vite + Tailwind CSS](https://github.com/use-preset/tailwindcss) - Adds Tailwind CSS to a fresh Vite application, or creates the Vite application.
- [Preset](https://github.com/use-preset/preset) - Scaffolds a preset.
- [ESLint](https://github.com/mloberg/preset-eslint) - Adds ESLint to a project.

> - [Next.js + ESLint + Prettier](https://github.com/skydiver/preset-nextjs/) - Install Next.js & adds support for ESLint + Prettier

Reviewer: ```suggestion
- [Next.js + ESLint + Prettier](https://github.com/skydiver/preset-nextjs/) - Installs Next.js with ESLint and Prettier.
```
